korlibs-logger/src@mingw/korlibs/logger/Logger.mingw.kt

```diff
@@ -0,0 +1,42 @@
+@file:OptIn(ExperimentalForeignApi::class)
+
+package korlibs.logger
+
+import kotlinx.cinterop.*
+import platform.windows.*
+
+actual object Console : BaseConsole()
+
+actual object DefaultLogOutput : Logger.Output {
+    override fun output(logger: Logger, level: Logger.Level, msg: Any?) = Logger.ConsoleLogOutput.output(logger, level, msg)
+}
+
+internal actual val miniEnvironmentVariables: Map<String, String> by lazy { getEnvs() }
+
+private fun readStringsz(ptr: CPointer<WCHARVar>?): List<String> {
```

Comment:
Is the z in the function name on purpose?

Reply:
Yes. The stringz is typically from C-languags world. It is a string that ends with a '\0' byte. Typically in C strings strings work like that instead of having the size attached, they just end by '\0'